tests/baselines/reference/user/webpack.log

```diff
@@ -4,27 +4,11 @@ lib/ContextModule.js(436,34): error TS2341: Property 'moduleGraph' is private an
 lib/ContextModule.js(465,34): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
 lib/ContextModule.js(767,34): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
 lib/Dependency.js(139,29): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
-lib/ExternalModule.js(284,54): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
+lib/ExternalModule.js(285,54): error TS2341: Property 'moduleGraph' is private and only accessible within class 'ChunkGraph'.
+lib/FileSystemInfo.js(362,4): error TS2345: Argument of type '({ type, context, path, expected }: { type: any; context: any; path: any; expected: any; }, callback: any) => void' is not assignable to parameter of type 'AsyncFunction<{ type: number; path: string; context?: string; expected?: string; }, Error>'.
```

Comment:
this looks legitimately incorrect afaict: the order of parameters is backward, and is likely caught by our improvements in destructuring.